### ISSUE TITLE
Default text for the text box

### ISSUE THREAD

**Issue by fmbfla** [adafruit/io-issues#49]
All i would like to do is when I create the text box, I would like to have the option to put a default text message for when the IO device is NOT ACTIVE.
 I have the simple code to send for Active but if the device connection goes down I don't know how to send the message "Not Active" when the device connection has been broken or if the device has hung.
The stream block doesn't hold the last RX data on the page load or refresh. 
Basically, 
So if after a predefined time if AIO has not received data the default message for the Text Block would be displayed. as NOT ACTIVE.
Thanks for all you do :-)

Drew

**Comment by toddtreece** [comment]
@fmbfla which device and library are you using? are you using MQTT or the HTTP REST API?

**Comment by jwcooper** [comment]
I'll set an option for the stream block to load the last values of any feeds.

We also have a feature coming soon-ish that will be configurable to alert you when a feed has gone 'offline', and isn't receiving any data. So, if after {15,30,60} minutes one of your devices crashes or loses connection, you'd get an email or some type of alert.

**Comment by fmbfla** [comment]
Todd,
I'm using MQTT an Arduino UNO, with an ADA cc3K BoB, BMP180, HDC 100x, and a INA 219.
I am loading a Modified group_sensor Sketch.

Coop, That's great to hear and will be useful.

However by .having the Txt Block to load the last value it would show "active" even while it was "In Active". I would like to add the word "Inactive" to the Default Txt Block message so that every time I send DATA I can send the word "Active" and therefore show that the data is "fresh" and not "stale" understand me ? LOL.
Thanks again Guy's ...
Drew.

**Comment by toddtreece** [comment]
@fmbfla I think what you are looking for is a combo of MQTT's [Last Will](http://www.hivemq.com/blog/mqtt-essentials-part-9-last-will-and-testament) feature for connection drops, and a standard publish to that topic to change state when your client cleanly disconnects. I'll see if I can get last will implemented in the MQTT library.

**Comment by jwcooper** [comment]
Thanks for the feedback. We're tracking this internally now as well. Further issues or enhancement requests can go to our forum as we're consolidating our support and issues: https://forums.adafruit.com/viewforum.php?f=56
